### ISSUE TITLE
Only see black tiles when exporting package.

### ISSUE THREAD

**Issue by ryrichard** [mapbox/mapbox-unity-sdk#1777]
Unity version: 2019.4.1f1
Mapbox SDK version: 2.1.1
Platform Building: iOS
I am just trying to get the LocationBased game to work on a fresh build. I managed to get the unitypackage to work with the sdk developer. It even worked when building to my iphone. The issue is when I tried to export it as a package and added it to a fresh build. While it works on the editor, on my iphone all I see is black tiles. It is not a permission issue because it displays my coordinates at the bottom. Its not an internet issue because I already successfully built it with the developer version. Any suggestion would be helpful.

**Comment by MRKDaGods** [comment]
What shader are you using for the Tile Material?

**Comment by ryrichard** [comment]
Terrain Material

**Comment by ryrichard** [comment]
So I managed to fix it. All I did was look through my fresh build's player settings and the sdk developer's player settings. Noticed there was a lot of differences in there, but I only changes I made was turning Auto Graphics API on and adding "CROSS_PLATFORM_INPUT;MOBILE_INPUT" to scripting define symbols. Don't know if these changes would have been enough, don't remember if I made some other small changes. Anyways I got it to work.
